kiali.go

```diff
@@ -90,6 +90,21 @@ func main() {
 	status.Put(status.CoreCommitHash, commitHash)
 	status.Put(status.ContainerVersion, determineContainerVersion(version))
 
+	authentication.InitializeAuthenticationController(config.Get().Auth.Strategy)
```

Comment:
since this is being done here, delete the dup call on 108

Reply:
Done.